### ISSUE TITLE
Updating all apps, when not commanded so

### ISSUE THREAD

**Issue by Olf0** [mentaljam/harbour-storeman#96]
Storeman seems to erroneously update all apps listed in *Installed applications: Update available*, when:
- updating the first (topmost) app listed there per *long press -> Update*
- uninstalling any app per *long press -> Remove*, which is listed in the *Update available* section

In all other cases individually updating apps per *long press -> Update* in *Installed applications: Update available* seems to work fine.
Side note: Updating all apps listed there per *Update all* in the pulley menu sure works fine.

Observed with Storeman 0.1.5

P.S.: Loosely related to issue #92.

**Comment by mentaljam** [comment]
Yes it's an annoying known issue. Currently I'm rewriting (again) the core part of Storeman. I hope it should fix this issue.

**Comment by Pohli** [comment]
It also happens if updating per pulley menu from app details view.

**Comment by mentaljam** [comment]
It seems to be an issue of PackageKit

**Comment by Olf0** [comment]
That would be a bit unfortunate, as we would have to wait for the PackageKit maintainers **and** Jolla in order to have this issue resolved.
Did you find a corresponding issue report for PackageKit or is it trying to find a solution for so long, what made you assuming a PackageKit bug?

**Comment by mentaljam** [comment]
> what made you assuming a PackageKit bug

The issue is actual for pkcon as well as for Storeman. Also (I understand it's not a good argument, but) I couldn't find a bug in the Storeman's code.

> Did you find a corresponding issue report for PackageKit or is it trying to find a solution for so long

Sorry, I had too much life events the last half a year. I just can't develop Storeman right now.

**Comment by Olf0** [comment]
That absolutely is O.K., real life should always come first!

So I might try to find a similar PackageKit issue and its status, when I have some time for that.
I would be happy, if @Pohli or someone else interested could contribute to that.

**Comment by szopin** [comment]
Also happens when installing a new app

**Comment by Olf0** [comment]
> Also happens when installing a new app

Yes, updating or freshly installing makes no difference for this PackageKit misbehaviour.

But it does not always happen, even though often.

**Comment by Olf0** [comment]
This [also happens with the SailfishOS:Chum GUI app](https://github.com/sailfishos-chum/sailfishos-chum-gui/issues/67), which utilises PackageKit, too.

Hence this is quite clearly a bug in PackageKit which should be filed there.
